api/pom.xml

```diff
@@ -13,7 +13,7 @@
    <parent>
       <groupId>org.openmrs</groupId>
       <artifactId>openmrs</artifactId>
-      <version>2.2.0-SNAPSHOT</version>
+      <version>2.2.0-passwordreset</version>
```

Comment:
we definitely want to revert the version change except if we wanna maintain this in a separate branch

Reply:
Yes, good catch!